aspnetcore/tutorials/razor-pages/razor-pages-start.md

```diff
@@ -1,25 +1,164 @@
 ---
 title: "Tutorial: Get started with Razor Pages in ASP.NET Core"
 author: rick-anderson
-description: This series of tutorials shows how to use Razor Pages in ASP.NET Core. Learn how to create a model, generate code for Razor pages, use Entity Framework Core and SQL Server for data access, add search functionality, add input validation, and use migrations to update the model.
+description: This is the first tutorial of a series that teaches the basics of building an ASP.NET Core Razor Pages web app.
 ms.author: riande
-ms.date: 11/12/2019
-no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+ms.date: 09/15/2020
+no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/razor-pages/razor-pages-start
 ---
 
 # Tutorial: Get started with Razor Pages in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
 This is the first tutorial of a series that teaches the basics of building an ASP.NET Core Razor Pages web app.
 
-[!INCLUDE[](~/includes/advancedRP.md)]
+For a more advanced introduction aimed at developers who are familiar with controllers and views, see [Introduction to Razor Pages](xref:razor-pages/index).
 
 At the end of the series, you'll have an app that manages a database of movies.  
 
-[!INCLUDE[View or download sample code](~/includes/rp/download.md)]
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30) ([how to download](xref:index#how-to-download-a-sample)).
+
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Create a Razor Pages web app.
+> * Run the app.
+> * Examine the project files.
+
+At the end of this tutorial, you'll have a working Razor Pages web app that you'll enhance in later tutorials.
+
+![Home or Index page](razor-pages-start/_static/5/home5.png)
+
+## Prerequisites
+
+# [Visual Studio](#tab/visual-studio)
+
+[!INCLUDE[](~/includes/net-core-prereqs-vs-5.0.md)]
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/net-core-prereqs-vsc-5.0.md)]
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+[!INCLUDE[](~/includes/net-core-prereqs-mac-5.0.md)]
+
+---
+
+## Create a Razor Pages web app
+
+# [Visual Studio](#tab/visual-studio)
+
+1. From the Visual Studio **File** menu, select **New** > **Project**.
```

Comment:
That's not how the current VS works.  The following image shows the default. cc @scottaddie 

![image](https://user-images.githubusercontent.com/3605364/98317614-e436be80-1f80-11eb-8127-67c50414b5bf.png)

Reply:
@wadepickett  please fix this ASAP and provide a link to https://docs.microsoft.com/en-us/visualstudio/ide/create-new-project

Reply:
@Rick-Anderson, yes, I should had something in there instead like: 
From the Visual Studio start window, choose **Create a new project**.  For more information, see [Create a new project in Visual Studio](https://docs.microsoft.com/visualstudio/ide/create-new-project).

The link does feel a bit like a distraction, but I am adding it.

Reply:
That's not the point. These are the really old instructions that are totally different now.